### DISCUSSION TITLE
[Bug]: ComposedModal has a mouse up event that's causing issues

### DISCUSSION THREAD

**Discussion by clairek20** [carbon-design-system/carbon#10154] [Help] [answered]
### Package

carbon-components

### Browser

Chrome

### Package version

Most recent

### Description

Modal will close when mousing up outside modal

Open https://carbon-components-react.netlify.app/?path=/docs/components-composedmodal--default
hightlight textbox to delete text and mouse up outside of modal - modal will close

### CodeSandbox example

https://codesandbox.io/s/github/carbon-design-system/carbon/tree/main/packages/react/examples/codesandbox

### Steps to reproduce

Open https://carbon-components-react.netlify.app/?path=/docs/components-composedmodal--default
hightlight textbox to delete text and mouse up outside of modal - modal will close

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by tay1orjones** [comment, marked as answer]
Hi @clairek20, have you tried setting the `preventCloseOnClickOutside` prop to `true`? In the [playground story](https://carbon-components-react.netlify.app/?path=/story/components-composedmodal--playground) you can turn it on and off via the knobs pane. When checked, it prevents the modal from closing when clicking outside of the modal.

**Reply by tay1orjones** [reply]
[Internal slack discussion](https://ibm-studios.slack.com/archives/C2K6RFJ1G/p1637600928421200?thread_ts=1637591761.410600&cid=C2K6RFJ1G) confirming this as a resolution to the issue.
